### PR TITLE
Add basic dmaker.fan.1c support

### PR DESCRIPTION
**Pull request by syssi**

### CODE REVIEW

**Comment by override80** [comment]
Hi @rytilahti , is there anything that still needs to be done to merge this?

File: miio/fan_miot.py [hunk 1]
```diff
@@ -141,6 +153,58 @@ def child_lock(self) -> bool:
...
+        return self.data["swing_mode"]
+
+    @property
+    def delay_off_countdown(self) -> int:
+        """Countdown until turning off in seconds."""
+        return self.data["power_off_time"]
+
+    @property
+    def led(self) -> bool:
+        """True if LED is turned on, if available."""
```
Comment (rytilahti) [inline miio/fan_miot.py:194]: Considering "if available", should this use `self.data.get("light")` and be typed as `Optional[bool]`?

**Review by rytilahti — COMMENTED** [review]
LGTM, just a couple of minor nitpicks.

**Comment by rytilahti** [comment]
@override80 I hadn't done a review on this as @syssi hasn't requested one and there's a todo in the PR description. He is also a co-maintainer of this project, so he doesn't really need my approval for merges (even when I think it's useful to get some eyeballs on the code pre-merge) :-)

@syssi do you think we could merge this soonish, so that we can prepare a release? I'd like to get a new release out soon to fix those AQI issues that are piling up on homeassistant issue tracker.

**Comment by syssi** [comment]
I will try to write the tests this weekend.

**Comment by override80** [comment]
> @override80 I hadn't done a review on this as @syssi hasn't requested one and there's a todo in the PR description. He is also a co-maintainer of this project, so he doesn't really need my approval for merges (even when I think it's useful to get some eyeballs on the code pre-merge) :-)
> 
Ouch OK!  I didn't know 😅

File: miio/fan_miot.py [hunk 2]
```diff
@@ -141,6 +153,58 @@ def child_lock(self) -> bool:
         return self.data["child_lock"]
 
 
+class FanStatusC1(DeviceStatus):
+    """Container for status reports for Xiaomi Mi Smart Pedestal Fan DMaker C1."""
```
[the remark this answers is not in the document]
Comment (syssi) [reply miio/fan_miot.py]: Added.

**Pull request merged**